### PR TITLE
Release/snowplow fractribution/0.2.0

### PR DESCRIPTION
**Pull request by agnes-kiss**
## Description & motivation
This release adds support for Databricks and BigQuery. From now users are allowed to filter unwanted channels as well. Under the hood we added an integration testing suite and some automations with Github actions.

## Features
Add support for Databricks and BigQuery
Add release and pages actions (#5)
Allow user to filter unwanted channels (#3)
Add versioning for Docker

## Checklist
- [x] I have verified that these changes work locally
- [x] I have updated the README.md (if applicable)
- [x] I have added tests & descriptions to my models (and macros if applicable)
- [x] I have raised a [documentation](https://github.com/snowplow/documentation/pull/309/files) PR

## Release Only Checklist
- [x] I have updated the version number in all relevant places
- [x] I have updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,23 @@
...
+
+## Features
+Add support for Databricks and BigQuery
+Add release and pages actions (#5)
+Allow user to filter unwanted channels (#3)
+Add versioning for Docker
+
+## 🚨 Breaking Changes 🚨
+Variable names have changed, please align the new ones found in the dbt_project.yml file and the ones defined in your own project's yml file that you used to overwrite the default values. Please note that
+`snowplow__path_transforms` variable is a dictionary instead of an array.
```
Comment (rlh1994) [inline CHANGELOG]: We also now require dbt 1.3 I think?

File: README.md [hunk 2]
```diff
@@ -16,11 +16,12 @@ Please refer to the [doc site](https://docs.snowplow.io/docs/modeling-your-data/
...
 
-The snowplow-fractribution v0.1.0 package currently supports Snowflake.
+The snowplow-fractribution v0.2.0 package currently supports Snowflake.
 
-|      Warehouse     |    dbt versions     | snowplow-fractribution version |
-| :----------------: | :-----------------: | :----------------------------: |
-|      Snowflake     |  >=1.0.0 to <2.0.0  |             0.1.0              |
+|      Warehouse                           |    dbt versions     | snowplow-fractribution version |
+| :--------------------------------------: | :-----------------: | :----------------------------: |
+|      Snowflake, BigQuery, Databricks     |  >=1.0.0 to <2.0.0  |             0.2.0              |
```
Comment (rlh1994) [inline README.md]: dbt version needs incrementing here as well

**Review by rlh1994 — CHANGES_REQUESTED** [review]
LGTM, just need to highlight we now require 1.3 dbt version, also needs changing in the docs PR as well, other than that it's good to go for tomorrow!

File: CHANGELOG (hunk 1 above)
Comment (agnes-kiss) [reply CHANGELOG]: Oh good point, I missed to add that bit eventually.

**Review by rlh1994 — APPROVED** [review]
Ready to go tomorrow!

File: README.md [hunk 3]
```diff
@@ -67,8 +67,7 @@ The tables produced by the Python script are the following:
...
 5. Configure the channel_spend macro to query your own spend data if you do not wish to use the default values.
-6. Configure environment variables to be used by the python file specific to your warehouse (either `main_snowplow_bigquery.py`, `main_snowplow_databricks.py` or `main_snowplow_snowflake.py`) to enable you to connect.
-7. Overwrite default variables provided by the package in your dbt_project.yml file, if necessary. E.g.: make sure your `snowplow__page_views_source` and `snowplow__conversions_source` are aligned to what is available in your warehouse, and update snowplow__conversion_window_start_date and snowplow__conversion_window_end_date if you don't want the default of the last 30 days.​
+6. Overwrite default variables provided by the package in your dbt_project.yml file, if necessary. E.g.: make sure your `snowplow__page_views_source` and `snowplow__conversions_source` are aligned to what is available in your warehouse, and update snowplow__conversion_window_start_date and snowplow__conversion_window_end_date if you don't want the default of the last 30 days.​
```
Comment (rlh1994) [inline README.md]: Could you make the other variable names in here in backticks as well please?

**Review by rlh1994 — APPROVED** [review]
Looks good to me with just one nitpick, can you squash this into the appropriate commit as well then it's good to go!

**Review by rlh1994 — APPROVED** [review]
LGTM!

**Pull request merged**